.circleci/config.yml

```diff
@@ -102,7 +102,7 @@ jobs:
     docker:
       - image: circleci/node:lts
       - image: trufflesuite/ganache-cli
-        command: ganache-cli -i 1234 -l 9000000
+        command: ganache-cli -i 1234 -l 9000000 -f https://mainnet.infura.io/v3/e34138b2db5b496ab5cc52319d2f0299
```

Comment:
what Infura key is this? do we really want to fork ganache in CLI?

Reply:
I can understand that you can potentially need this for the 1inch tests but what is the impact on other tests in CI? What would the outcome be if this key exceeds the quota? would CI fail until the next quota interval?

Reply:
The project id was the (old) ethers-js default infura key.

It should have 0 impact on other tests in the CI as the CI is testing freshly deployed contracts, not referencing existing onces.

Reply:
I see, that makes sence. so no slow down if we dont need the functionality of main-net forking. system continues as is right now and _iff_ you need forking then you have access to it.